cli/ast.rs

```diff
@@ -1,13 +1,15 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
 use crate::media_type::MediaType;
+use crate::module_graph2::Graph2;
```

Comment:
That makes a circular dep between `module_graph2` and `ast`. Could it be removed?

Reply:
Let me take a look at it again...  It is how to cleanly impl the bundle loader in `module_graph2` without dragging all of ast in.